documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -36,6 +37,14 @@ export default {
 
 Path to your custom `wrangler.toml` or `wrangler.json` config file.
 
+### handlers
+
+Path to a file with additional [handlers](https://developers.cloudflare.com/workers/runtime-apis/handlers/) and (optionally) [Durable Objects](https://developers.cloudflare.com/durable-objects/) to be exported from the file the adapter generates. This allows you to, for example, include handlers for scheduled or queue triggers alongside the fetch handler your SvelteKit app.
+
```

Comment:
Can we add a code example here or what the `handlers` file should look like? For example:
```js
/// file: src/handlers.js

// export your durable objects here

export default {
  // handlers go here
}
```